### PR TITLE
Dbl click event emit

### PR DESCRIPTION
**Pull request by kepoling**

### CODE REVIEW

**Comment by wvengen** [comment]
:+1: will merge and release a new version

File: d3.chart.sankey.js [hunk 1]
```diff
@@ -122,6 +122,7 @@ return /******/ (function(modules) { // webpackBootstrap
 						this.on("mouseover",  function(e) { chart.trigger("link:mouseover", e); });
 						this.on("mouseout",   function(e) { chart.trigger("link:mouseout",  e); });
 						this.on("click",      function(e) { chart.trigger("link:click",     e); });
+						this.on('dblclick',   function(e) { chart.trigger("link:dblclick",  e); });
```
Comment (wvengen) [inline d3.chart.sankey.js]: I suppose you meant double quotes here ... ;)

Comment (kepoling) [reply d3.chart.sankey.js]: You are correct :) will amend.

**Comment by wvengen** [comment]
Forgot documentation in README, added in 4fae404.

**Pull request merged**